shotover-proxy/src/transforms/redis/sink_cluster.rs

```diff
@@ -529,10 +537,20 @@ impl SlotMap {
 #[derive(Debug, Clone, Copy)]
 pub enum RoutingInfo {
     AllNodes,
-    AllMasters,
+    AllMasters(ResponseJoin),
     Random,
     Slot(u16),
     Other(Command),
+    Unsupported,
```

Comment:
Does `Unsupported` represent an error getting a `RoutingInfo` struct for the message and we can't route it?

Reply:
It represents a command that the cluster sink does not support.
There are certain commands that there is no sensible way for shotover to handle here.